nodes/parachain/src/chain_spec/polimec.rs

```diff
@@ -180,7 +180,7 @@ pub fn get_rococo_chain_spec() -> GenericChainSpec {
 }
 
 fn base_testnet_genesis(
-	stakers: Vec<(AccountId, Option<AccountId>, Balance)>,
+	#[allow(unused)] stakers: Vec<(AccountId, Option<AccountId>, Balance)>,
```

Comment:
Why is this needed?

Reply:
because compiling with runtime-benchmarks gives a warning